src/doc/trpl/structs.md

```diff
@@ -47,3 +54,36 @@ fn main() {
 ```
 
 This will print `The point is at (5, 0)`.
+
+Rust does not support mutability at the field level, so you cannot write
```

Comment:
I would argue that internally mutable values can be thought of as immutable. But yes, the language here is hard.